x-pack/plugins/stack_connectors/public/connector_types/lib/test_utils.tsx

```diff
@@ -71,12 +74,12 @@ const FormTestProviderComponent: React.FC<FormTestProviderProps> = ({
   }, [onSubmit, submit]);
 
   return (
-    <I18nProvider>
```

Comment:
The i18n context provider is not explicitly needed since KibnaRenderContextProvider provides it